uid-pages/layout-bonita/src/assets/json/localization.json

```diff
@@ -161,52 +159,50 @@
-    "Center": "",
-    "Increase indent": "",
-    "Decrease indent": "",
-    "Clear formatting": "",
-    "Please enter an image URL to insert": "",
-    "Insert image": "",
-    "Insert video": "",
-    "Please enter a youtube URL to embed": "",
-    "Insert / edit link": "",
-    "Please enter a URL to insert": "",
-    "Relink": "",
-    "Unlink": "",
-    "Open in New Window": "",
-    "Display words Count": "",
-    "Display characters Count": "",
-    "The requested content cannot be loaded.<br />Please try again later.": "El contenido solicitado no puede mostrarse.",
-    "User details": "",
-    "Username": "",
-    "Language": "",
-    "Select a language": "",
-    "Select a profile": "",
-    "Logout": "",
-    "Save": "",
-    "Cancel": "",
-    "Application List": "",
-    "All": "",
-    "Search on name, version or URL": "",
-    "No application available using these filters": "",
-    "Version": "",
-    "Close": "",
+    "Toggle the text area to HTML source code / Rich text area": "Cambiar la zona de texto por código fuente HTML / Área de texto enriquecido",
+    "Heading ": "Título",
+    "Paragraph": "Parráfo",
+    "Preformatted text": "Texto preformateado",
+    "Unordered List": "Lista sin ordenar",
+    "Ordered List": "Lista ordenada",
+    "Quote/unquote selection or paragraph": "Citar / no citar selección o párrafo",
+    "Undo": "Deshacer",
+    "Redo": "Rehacer",
+    "Bold": "Negrita",
+    "Italic": "Itálico",
+    "Underline": "Subrayado",
+    "Strikethrough": "Tachado",
+    "Align text left": "Alinear texto a la izquierda",
+    "Align text right": "Alinear texto a la derecha",
+    "Justify text": "Justifica el texto",
+    "Center": "Centrar",
+    "Increase indent": "Aumentar sangría",
+    "Decrease indent": "Disminuir sangría",
+    "Clear formatting": "Borrar el formato",
+    "Please enter an image URL to insert": "Por favor ingrese una URL de imagen para insertar",
+    "Insert image": "Insertar imagen",
+    "Insert video": "Insertar video",
+    "Please enter a youtube URL to embed": "Por favor, introduzca una URL de youtube para incrustar",
+    "Insert / edit link": "Insertar / editar enlace",
+    "Please enter a URL to insert": "Por favor ingrese una URL para insertar",
+    "Relink": "Rehacer el enlace",
+    "Unlink": "Deshacer el enlace",
+    "Open in New Window": "Abrir en una nueva ventana",
+    "Display words Count": "Mostrar el número de palabras",
+    "Display characters Count": "Mostrar el número de carácteres",
+    "The requested content cannot be loaded.<br />Please try again later.": "El contenido solicitado no puede mostrarse.<br />Por favor, intente otra vez más tarde",
+    "User details": "Detailles del usuario",
+    "Username": "Nombre de usuario",
+    "Language": "Idioma",
+    "Select a language": "Seleccione un idioma",
+    "Logout": "Desconectar",
+    "Save": "Guardar",
+    "Cancel": "Cancelar",
+    "Application List": "Lista de applicaciones",
+    "All": "Todas",
```

Comment:
Since this is used to list all the profiles it should be "Todos", I think

Reply:
Both are apparently valid, depending if it's feminine or masculine gender.